app/models/course.rb

```diff
@@ -78,4 +100,15 @@ def duration(format = "mins")
     Duration.minutes_str(total, format)
   end
 
+  def set_pub_date
+    pub_date = DateTime.now.strftime("%m/%d/%Y") if pub_status == "P"
+  end
+
+  def update_pub_date(new_pub_status)
+    if new_pub_status == "P"
+      pub_date = DateTime.now.strftime("%m/%d/%Y")
```

Comment:
Same problem as the other line. Converting it to Ameican format confuses Rails when it parses it into the International format.